model/trace/instrumentation/gcp-client.yml

```diff
@@ -0,0 +1,43 @@
+groups:
+  - id: gcp.client.scope
+    type: scope
+    brief: >
+      The `gcp.client` conventions are intended to be used by Google Cloud
+      client libraries (e.g. https://github.com/googleapis/* and other
+      Google-owned/Google-provided client libraries for Google Cloud) to
+      identify such clients and to track their operations and performance.
+    prefix: gcp.client
+    stability: experimental
+    attributes:
+      - id: gcp.client.service
+        type: string
+        stability: experimental
+        requirement_level: required
+        brief: >
+          Short name of the GCP service for which the library is a client.
+          All variants of the client across programming languages should use an
+          identical value for this property. The short name should be intuitive
```

Comment:
You say that these all need to be identical. I agree. I think using the endpoint name will be deterministic. (e.g. `foo` from `foo.googleapis.com`).

Context from the client library perspective:
I think the short names of clients vary across the client library languages. At least in C++ we did a lot of ad-hoc naming. `google-cloud-X` will not be consistent, so let's not mention it.

Reply:
Thanks! I changed the guidance so that the derivation is done solely from the API domain name.